test/kafka-resumption/sink-queue-full/setup.td

```diff
@@ -18,14 +18,20 @@ $ kafka-create-topic topic=largeinput
 $ kafka-ingest topic=largeinput format=bytes
 A,B,0
 
-> CREATE CONNECTION IF NOT EXISTS kafka_conn TO KAFKA (BROKER '${testdrive.kafka-addr}', SECURITY PROTOCOL PLAINTEXT);
+> CREATE CONNECTION IF NOT EXISTS kafka_conn1 TO KAFKA (BROKER '${testdrive.kafka-addr}', SECURITY PROTOCOL PLAINTEXT);
+> CREATE CONNECTION IF NOT EXISTS kafka_conn2 TO KAFKA (BROKER 'toxiproxy:9093', SECURITY PROTOCOL PLAINTEXT);
 
 # The source intentionally does not go through toxiproxy.
```

Comment:
@def- I think this test needs to remain as is according to this comment

Reply:
We could make this more obvious by moving the comment above the `CREATE CONNECTION` 😀